src/components/button/button.tsx

```diff
@@ -11,12 +11,12 @@ const icons = {
 	metamask
 };
 
-type ColorType = 'default' | 'error' | 'warning' | 'icon';
+type ColorType = 'default' | 'error' | 'warning' | 'icon' | 'selected';
```

Comment:
`selected` is not a color prop, it has to be solved with the logic of the button